src/content/testing/index.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Okteto Test
+description: In this section, we'll see how you can use Test Containers on Okteto
+---
+
+# Welcome to Okteto Test
+
+Okteto Test is a new feature designed to seamlessly integrate testing into your development workflow on the Okteto platform. Whether you're running unit tests or end-to-end (E2E) tests, Okteto Test ensures an efficient testing experience before even making a pull request.
+
+## Benefits of Okteto Test
+
+### 1. Seamless Integration
+Developers can execute tests in the same production-like Kubernetes environment where their code runs.
+This ensures consistency and reduces configuration overhead.
+
+### 2. Simplified Configuration
+The new `test` section in the [Okteto manifest](reference/okteto-manifest.mdx#test-object-optional) supports multiple Test Containers, allowing you to define your test environment with minimal YAML configuration.
+This eliminates the need for complex setups and advanced Kubernetes knowledge.
+
+### 3. Faster Feedback Loop
+Okteto Test leverages Buildkit's built-in capabilities to run only the tests affected by code changes.
+You can quickly identify and fix issues, especially in monorepos, significantly reducing the feedback loop for a development cycle.
+
+
+## Getting Started with Okteto Test
+
+View our [Getting Started with Okteto Test Guide](testing/getting-started-test.mdx) to learn how to configure Test Containers.
+
+### Okteto Test Basics
```

Comment:
@pchico83 What do you think of this section? Worth having or just point to the guide?

Reply:
I think I would go with the guide, but not hard feelings :-)